functions.js

```diff
@@ -0,0 +1,52 @@
+const listContainer = document.querySelector('.list-container');
+const itemsContainer = document.querySelector('.list-container');
+
+export default class Collection {
```

Comment:
I think 🤔 the class name should be something more meaningful like ToDoList or ToDoCollection. it makes more sense, collection sounds more general.